pkg/runtime/ref/injector.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 Layotto Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ref
+
+import (
+	"github.com/dapr/components-contrib/secretstores"
+
+	"mosn.io/layotto/components/ref"
+)
+
+type DefaultInjector struct {
+	Container RefContainer
+}
+
+//InjectSecretRef  inject secret to metaData
+// TODO: permission control
```

Comment:
I added a "TODO" comment here. We need to add some permission control logic here, in the future in some other PR